internal/jsre/ethereum_js.go

```diff
@@ -3926,6 +3943,7 @@ module.exports = {
     inputCallFormatter: inputCallFormatter,
     inputTransactionFormatter: inputTransactionFormatter,
     inputAddressFormatter: inputAddressFormatter,
+	inputOptionalBoolFormatter: inputOptionalBoolFormatter,
```

Comment:
Small whitespace problem. Code around uses <whitespace> but this line uses <tab>.